install_config/imagestreams_templates.adoc

```diff
@@ -171,10 +173,35 @@ based images:
 $ oc create -f $IMAGESTREAMDIR/image-streams-centos7.json -n openshift
 ----
 
-It is not possible to create both the CentOS and RHEL sets of image streams
-because they use the same names. If you desire to have both sets of image
-streams available to users, either create one set in a different project, or
-edit one of the files and modify the image stream names to make them unique.
+Creating both the CentOS and RHEL sets of image streams is not possible, because
+they use the same names. To have both sets of image streams available to users,
+either create one set in a different project, or edit one of the files and
+modify the image stream names to make them unique.
+endif::[]
+
+ifdef::openshift-dedicated[]
+You can view all default image streams and their definitions using the CLI. To
+get a list of the current objects in the global *openshift* project:
+
+----
+$ oc get imagestreams -n openshift
+----
+
+To view or modify the definition for a specific image stream:
+
+----
+$ oc edit imagestream <imagestream_name> -n openshift
+----
+
+If you define your own image stream, first save it to a file, then create it in
+the *openshift* project using the CLI:
+
+----
+$ oc create -f <file_name> -n openshift
+----
+
+The image stream is then made available to all users across the cluster.
```

Comment:
I think "then" sounds strange.  How about "This makes the image stream available to ...".